src/gui/wizard/owncloudconnectionmethoddialog.cpp

```diff
@@ -29,8 +29,11 @@ OwncloudConnectionMethodDialog::OwncloudConnectionMethodDialog(QWidget *parent)
     connect(ui->btnClientSideTLS, SIGNAL(clicked(bool)), this, SLOT(returnClientSideTLS()));
     connect(ui->btnBack, SIGNAL(clicked(bool)), this, SLOT(returnBack()));
 
-    // DM: TLS Client Cert GUI support disabled for now
+
+#if QT_VERSION < QT_VERSION_CHECK(5, 4, 0)
```

Comment:
No, for old versions we want to hide this button to not allow the user to configure anything.
This is based on 3dd6bcc3233cd7c4ad88eecb43af726e11869601